### PR TITLE
fix panic on trigger drop

### PR DESCRIPTION
**Pull request by jacekchmiel**
Hey, I've found a serious bug in my use case. This perfectly fixes it. Merge it if you'd like so or if any changes are needed I'd be happy to help.

More information in commit message.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -160,7 +160,9 @@ impl Trigger {
 impl Drop for Trigger {
     fn drop(&mut self) {
         if let Some(tx) = self.0.take() {
-            tx.send(()).unwrap();
+            // Send may fail when all associated rx'es are dropped already
+            // so code here cannot panic on error
+            tx.send(()).unwrap_or_else(|_| ())
```
Comment (jonhoo) [inline src/lib.rs]: Can you make this the more idiomatic
```rust
let _ = tx.send();
```
instead?

**Comment by jonhoo** [comment]
Ah, good catch! I left a minor inline comment, but am happy to merge and release once that has been fixed! :tada:

**Comment by jacekchmiel** [comment]
Happy to help! Made the line more idiomatic, should be ready to merge. :beers:

**Comment by jonhoo** [comment]
Thanks! Published in 0.4.3!

**Pull request merged**